README.md

```diff
@@ -306,6 +307,19 @@ Some servers may support certain helpers and not others. The AEP and associated
 
 Similarly, matching behavior will be determined by each implementing system. For example, some systems may match people based on email address or other information.
 
+### Deviations from RESTful Behavior
+
+This section outlines areas where the expectations of the OSDI customer community differ from boilerplate RESTful behavior and specify best practice expected behavior.
+
+#### Merging Objects (hashes) and Arrays on update
+
+Numerous resources in OSDI contain embedded objects (hashes) and arrays. For example, on Person, Birthday and Custom Fields are Objects (hashes) and Postal Addresses is an array.
+
+When dealing with updates to these elements of resources, servers should merge rather than replace these elements.  This is more consistent with expected user behavior.  For example, sending a person_signup_helper, or a PUT on a person resource with only 1 address should not delete any existing addresses.  Similarly, including only 1 custom field, should not delete all existing custom fields on a resource.
+
+#### Deleting the full contents of an Object (hash) or Array
+In order to cause the deletion of the contents of an Object (hash) or Array, a request should include that element, but set its value to ````null````
```

Comment:
I think an example would be useful here, since there's a question in my mind about "addressing" the item to be deleted.  If a person has three addresses and the second is to be deleted, what should the request body look like?  Kind of along similar lines I'd rather that PUT requests have "overwrite" semantics, wrt the text above.